trunk/NDHMS/Routing/module_GW_baseflow.F

```diff
@@ -90,6 +95,9 @@ subroutine simp_gw_buck_nhd(           &
 !!!Initialize variables...
    net_perc = 0.
    qout_gwsubbas = 0.
+   if (bucket_loss .eq. 1) then
+      qloss_gwsubbas = 0.
```

Comment:
We really need a constants module...

Reply:
@jmccreight agreed, but 0 and 1 are the _least_ magic numbers...

Reply:
mere mortal numbers, especially if they are integers...